src/board.rs

```diff
@@ -199,9 +242,37 @@ impl Chessboard {
         '.'
     }
 
-    /// Forsyth–Edwards Notation Serializer
-    /// * `chessboard` - The chessboard position to be converted to a FEN.
-    /// # Return: FEN string representing the board's position.
+    /// Serializes a chessboard position into Forsyth–Edwards Notation (FEN).
+    ///
+    /// # Arguments
+    ///
+    /// - `chessboard`: The chessboard position to be converted to FEN.
+    ///
```

Comment:
this should not take any arguments, other than the assumed `&self` which doesn't need to be included in the docstring.